controllers/errorHandling.js

```diff
@@ -0,0 +1,4 @@
+// return response in case of error
+module.exports = (err, req, res, next) => {
+  res.status(err.statis || 500).json(err.message || 'Server error');
```

Comment:
Maybe I'm unaware of this syntax, but shouldn't this "read err.status" as opposed to "error.statis"? Either way, super cool that you have a file for error handling

Reply:
embarassing typo. yeah that will return undefined and end up being 500 all the time